starter-code/memory.js

```diff
@@ -51,12 +90,30 @@ $(document).ready(function(){
     html += '    name="' + pic.name + '">';
     html += '</div>';
     html += '<div class="front" ';
-    html += 'style="background: url(img/' + pic.img + '") no-repeat"';
+    html += 'style="background: url(img/' + pic.img + ') no-repeat"';
     html += '    name="'       + pic.name +  '">';
     html += '</div>';
     html += '</div>';
   });
 
+
   // Add all the divs to the HTML
   document.getElementById('memory_board').innerHTML = html;
+
+
+  // storing the clicked card as a variable to be called on the selectCard function (the one that handles the game mechanics and logic)
+
+$('.back').click(function() {
+    // $(this).attr("name");
+    cardName = $(this).attr("name");
+    memoryGame.selectCard(cardName);
+    if ( memoryGame.selectCard.length <= 2) {
```

Comment:
_memoryGame.selectCard_ is a function. Did you mean  _memoryGame.selectedCard_ ?